### ISSUE TITLE
Create a new release

### ISSUE THREAD

**Issue by rllola** [coinbase/rosetta-sdk-go#449]
Would it be possible to create a new release with the latest commit ? It looks like it is required for Rosetta CLI to pass the `check:spec`.

The commit required : https://github.com/coinbase/rosetta-sdk-go/commit/e03849ba430a9ba22997888c80acd537eef15fe3

**Comment by GeekArthur** [comment]
Yes, we will create releases for both Rosetta CLI and Rosetta SDK soon, usually the releases will be created monthly

**Comment by rllola** [comment]
Hey @GeekArthur 

I noticed in Rosetta CLI you added the check when no hash or index are specified in the partial block identifier. However because Rosetta CLI is using rosetta-sdk-go v0.8.1 it doesn't include the commit https://github.com/coinbase/rosetta-sdk-go/commit/e03849ba430a9ba22997888c80acd537eef15fe3

Because of that `check:spec` will always fail.

see the piece of code in v0.10.0
https://github.com/coinbase/rosetta-cli/blob/v0.10.0/cmd/check_spec.go#L306-L313

A release for rosetta-sdk-go and updating it in rosetta cli will fix it. Do you think it would be possible to make an early release just to fix that ? It would avoid us having to have our own fork until it is fixed.

**Comment by rllola** [comment]
@GeekArthur Any news about the release ?

**Comment by tac0turtle** [comment]
pinging about this, it would help us avoid a replace tag in our and our users go.mods

**Comment by GeekArthur** [comment]
Release [v0.8.2](https://github.com/coinbase/rosetta-sdk-go/releases/tag/v0.8.2) is published
